omise/omise.php

```diff
@@ -233,17 +252,18 @@ protected function getWebhooksEndpoint()
 
     public function hookDisplayOrderConfirmation($params)
     {
+
```

Comment:
@jonrandy there is 1 accidentally breakline here

Reply:
Really??

Reply:
Oh, so it's an intentionally breakline?